### ISSUE TITLE
missing `typedarray-to-buffer` after npm install...

### ISSUE THREAD

**Issue by gskapka** [oraclize/ethereum-examples#47]
Per title. Seems to be affecting all the Truffle examples recently.

**Comment by gskapka** [comment]
& also 'yaeti' after that too. 

Possibly need to make sure the package json only specifies exact versions of packages.

**Comment by riccardopersiani** [comment]
I wasn't able to replicate the error with the following:
```
Truffle v5.0.0-next.17 (core: 5.0.0-beta.1)
Solidity v0.5.0 (solc-js)
Node v10.6.0
Web3 v1.0.0-beta.36'
```

**Comment by riccardopersiani** [comment]
Seems to be an *error* related to **web3**. Will be solved in the web3 PR  [#2000](https://github.com/ethereum/web3.js/pull/2000) with web3@1.0.0-beta.28. Hopefully, Truffle will integrate it soon.

**Comment by riccardopersiani** [comment]
Fixed in PR #48
